Library/Navigation.swift

```diff
@@ -552,6 +552,11 @@ private func parsedParams(url: URL, fromTemplate template: String) -> RouteParam
     object[key] = value
   }
 
+  // If the deeplink opens the project page, track the url.
+  if urlComponents[0] == "projects" {
```

Comment:
Nit: is the URL guaranteed to have a path component? Otherwise you might want to do `urlComponents.first` instead.

Reply:
Yeah, it is right now but that might change in the future, so good call out! Fixed.